frontend/src/lib/components/Support/SupportForm.tsx

```diff
@@ -85,13 +100,45 @@ export function SupportForm(): JSX.Element | null {
                 <LemonSegmentedButton fullWidth options={SUPPORT_TICKET_OPTIONS} />
             </Field>
             <Field name="target_area" label="What area does this best relate to?">
-                <LemonSelect
-                    fullWidth
-                    options={Object.entries(TARGET_AREA_TO_NAME).map(([key, value]) => ({
-                        label: value,
-                        value: key,
-                    }))}
-                />
```

Comment:
can you start from latest on master 😊 

i've got an experiment running that relies on changes that aren't in this PR

Reply:
Already re-based this PR 😅